tests/util/model.py

```diff
@@ -57,7 +57,7 @@ class QuadraticWithUnitVariance(GaussianMarginal):
     """
     def __init__(self):
         self._model = CustomMeanWithUnitVariance(
-            lambda x: tf.reduce_sum(x ** 2, axis=1, keepdims=True)
+            lambda x: tf.reduce_sum(x ** 2, axis=-1, keepdims=True)
```

Comment:
the original function assumed a 2D tensor, while we need higher order ones

Reply:
you can resolve this if you are happy with this change

Reply:
i _think_ this is right, and I hope so, cos a lot of tests use it